test/test.xunit.assert/Asserts/EventAssertsTests.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Sdk;
+
+public class EventAssertsTests
+{
+    public class Raises_Generic
+    {
+        [Fact]
+        public static void ExpectedEventButCodeDoesNotRaise()
+        {
+            try
+            {
+                var x = new RaisingClass();
+                Assert.Raises<EventArgs>(h => x.Completed += h, h => x.Completed -= h, () => { });
```

Comment:
I would prefer to see these written with `Assert.Throws` (or `Record.Exception` + `Assert.IsType`) rather than try/catch, because as written, it's easier for them to pass false positives.